PhysicalTherapy/PhyscialTherapy/Client/src/app/therapist-routine-creation-screen/therapist-routine-creation-screen.component.ts

```diff
@@ -175,21 +175,21 @@ export class TherapistRoutineCreationScreenComponent implements OnInit {
     });
   }
 
-  handleFiles() {
+  handleFiles(routineId) {
     this.files.forEach((file) => {
-      this.uploadFile(file);
+      this.uploadFile(file, routineId);
     });
     this.files = [];
   }
 
-  uploadFile(file) {
+  uploadFile(file, routineId) {
     const fd = new FormData();
     fd.append('uploadFile', file, file.name);
     // We will need to have the upload URL location here
-    /* this.http.post('', fd)
+    this.http.post(`https://capstone-pt.azurewebsites.net`, fd)
```

Comment:
does it have to explicitly have the url as http.post there?/ is that how it works? other than that, it looks good to go

Reply:
Yeah, eventually it'll be something like /uploads/<routineId>. But this is just to see if it even works.